Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -155,7 +152,7 @@ void EmbeddedFrag::computeNbrsAndAng(unsigned int aid,
       anglePairs.emplace_back(ang, nbrPair);
     }
   }
-  anglePairs.sort(_anglComp);
+  anglePairs.sort([](auto pr1, auto pr2) { return pr1.first < pr2.first; });
```

Comment:
```suggestion
  anglePairs.sort([](const auto &pr1, const auto &pr2) { return pr1.first < pr2.first; });
```

Reply:
I'm leaving this as-is. Normally this would make perfect sense, but the arguments here are pairs of integers, which are small and cheap to copy. This is the kind of situation where the C++ core-guidelines recommend passing by value: